### ISSUE TITLE
The function Timex.local(Datetime) return the wrong result

### ISSUE THREAD

**Issue by Chicker** [bitwalker/timex#195]
Hello, all.

The function `Timex.local(Datetime)` return the wrong result. It should return the given date/time in the local timezone.
```
iex(36)> dt = Timex.now
#<DateTime(2016-07-16T12:12:24.396611Z Etc/UTC)>
iex(37)> Timex.local
#<DateTime(2016-07-16T17:12:30.492556+05:00 Asia/Yekaterinburg)>
iex(38)> Timex.local(dt)
#<DateTime(2016-07-16T12:12:24.396611Z Etc/UTC)>
```

The right result can be obtained with the this function:
```
iex(39)> Timex.Timezone.convert(dt, Timex.Timezone.local)
#<DateTime(2016-07-16T17:12:24.396611+05:00 Asia/Yekaterinburg)>
```
In the version 2.2.1 this bug is missing.

**Comment by bitwalker** [comment]
Oh interesting, it shifted it by a rather large number of extra microseconds, I'm not sure why that would've happened, but I will take a look and get that fixed. Thanks for the report!

**Comment by bitwalker** [comment]
Released in 3.0.4
